src/IO/Observer/ArrayComponentId.cpp

```diff
@@ -21,4 +21,9 @@ bool operator!=(const ArrayComponentId& lhs,
                 const ArrayComponentId& rhs) noexcept {
   return not(lhs == rhs);
 }
+
+std::ostream& operator<<(std::ostream& os,
+                         const ArrayComponentId& array_component_id) noexcept {
+  return os << std::hash<ArrayComponentId>{}(array_component_id);
```

Comment:
So I understand, why the {} before (array_component_id)?

Reply:
`std::hash` is a struct, so we need to instantiate it call it: https://en.cppreference.com/w/cpp/utility/hash